docs/asdf-astropy/example.rst

```diff
@@ -0,0 +1,119 @@
+.. _basic_example:
+
+=======
+Example
+=======
+
+
+In this example, we will show how to implement serialization for a new
+`~astropy.modeling.Model` object, but the basic principles apply to
+serialization of other ``astropy`` objects. As mentioned, adding a new object
+to ``asdf-astropy`` requires both a converter and a tagged-schema.
+
+All schemas for transforms are currently defined within the asdf-transform-schemas.
+Any new serializable transforms can have a corresponding new schema here. Let's
```

Comment:
"serializable models" might be more appropriate here.  And maybe "should have" instead of "can have"?